[all hosts] (Support) Following Teams support page format (#3863)
OfficeDev/office-js-docs-pr: docs/resources/resources-links-help.md

---
title: Office Add-ins additional resources
description: Useful resources about Office Add-ins.
ms.date: 02/22/2023
ms.localizationpriority: medium
---

# Office Add-ins additional resources

These resources provide additional information and support for developing Office Add-ins.

## Product support and service issues

Use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-critical issues** that need SLA-based support.

> | **Product support** | **Contact** |

Reviewer: I don't think you need to include the bold formatting in the top row as it's bolded by the platform

```suggestion
| Product support | Contact |
```

|:------------|:------------|
| **Microsoft 365 product issues and failures** | If you have a Premier support contract for Microsoft 365, visit the Microsoft 365 Admin Center and use the Support menu to [open a service request](https://admin.microsoft.com/). |
| **Azure help and support** | If you have a paid Azure subscription, visit the Azure Admin Center to [open a ticket](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportreq). |
| **General questions about Microsoft 365** | If you have general questions about Microsoft 365 or Office, submit your questions on [Microsoft 365 and Office Community](https://answers.microsoft.com/msoffice/forum).|

## Developer community help

Our community of developers use Stack Overflow, GitHub, and Microsoft Q&A to connect with other developers to ideate, get clarifications, and submit queries.

### Developer community forums

> Post your questions and help other community members by sharing and responding to Teams App Development questions.

Reviewer: ```suggestion
Post your questions and help other community members by sharing and responding to Office Add-in Development questions.
```

> | **Developer need**|   **Contact**  |

Reviewer: ```suggestion
| Developer need | Contact |
```

| **Office Add-in / Office JavaScript API questions** | <ul> <li> Post API questions to [Stack Overflow](https://stackoverflow.com/questions/tagged/office-js) using the `office-js` tag, and also include the `outlook-web-addins` tag if your question relates to Outlook add-ins. Please note that Stack Overflow has guidelines such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic; new users should visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask) for more details. </li> <li> You can also post questions using the `Office Development` tag on [Microsoft Q&A](/answers/tags/321/office-development)</li> |
| **Documentation gaps** | If the documentation is missing something that would help you understand how to create add-ins, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|

### Report issues

Submit issues and ask general questions related to add-ins, documentation, and samples using the community help channels.

> | **Issue type** | **Contact** |

Reviewer: ```suggestion
| Issue type | Contact |
```

|:----------------------|:------------|

> | **Office Add-ins or Office JavaScript API issues** | If you encounter an issue (bug) with the Office JavaScript API, please submit an issue to the [Office JavaScript APIs GitHub repository](https://github.com/officedev/office-js/issues)|
> |**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|

Reviewer: ```suggestion
| **Office Add-ins or Office JavaScript API issues** | If you encounter an issue (bug) with the Office JavaScript API, please submit an issue to the [Office JavaScript APIs GitHub repository](https://github.com/officedev/office-js/issues).|
|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues).|
```

| **Issues with code samples** | Submit issues with samples to the [Office Add-ins code samples](https://github.com/OfficeDev/Office-Add-in-samples) repository. |

### Documentation feedback

Submit documentation feedback or updates using the community help channels.

| **Feedback type** | **Contact** |
|:--------------------------|:--------------------------|

> |**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues)|
> |**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr)|

Reviewer: ```suggestion
|**Documentation issues**| If you encounter an issue (bug) with the documentation, please submit an issue to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr/issues).|
|**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the [Office Add-ins documentation GitHub repository](https://github.com/officedev/office-js-docs-pr).|
```

### Feature request and general help

Suggest a feature or vote up existing feature requests.

| **Support need** | **Contact** |
|:----------------------|:------------|
|**Office Add-ins or Office JavaScript API feature requests**| To submit a feature request for the Office JavaScript API, please post your idea to the [Microsoft 365 Developer Platform Tech Community](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform).|

> | **General questions** | Send general questions about the Office add-in platform to [Office Add-in Community Help](mailto:officeaddin. We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |

Reviewer: Is "mailto:officeaddin" the right destination for the Community Help link? Does it need to be `mailto:officeaddin@microsoft.com` or something like that?

```suggestion
| **General questions** | Send general questions about the Office Add-ins platform to [Office Add-ins Community Help](mailto:officeaddin). We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```

## External learning resources

The following books and classes provide add-ins information from a different perspective.

> |**Resource**|**More information**|

Reviewer: ```suggestion
|Resource|More information|
```
